include/zephyr/dt-bindings/clock/esp32c6_clock.h

```diff
@@ -44,44 +44,49 @@
-#define ESP32_PCNT_MODULE             10
-#define ESP32_SPI_MODULE              11
-#define ESP32_SPI2_MODULE             12
-#define ESP32_TWAI0_MODULE            13
-#define ESP32_TWAI1_MODULE            14
-#define ESP32_RNG_MODULE              15
-#define ESP32_RSA_MODULE              16
-#define ESP32_AES_MODULE              17
-#define ESP32_SHA_MODULE              18
-#define ESP32_ECC_MODULE              19
-#define ESP32_HMAC_MODULE             20
-#define ESP32_DS_MODULE               21
-#define ESP32_SDIO_SLAVE_MODULE       22
-#define ESP32_GDMA_MODULE             23
-#define ESP32_MCPWM0_MODULE           24
-#define ESP32_ETM_MODULE              25
-#define ESP32_PARLIO_MODULE           26
-#define ESP32_SYSTIMER_MODULE         27
-#define ESP32_SARADC_MODULE           28
-#define ESP32_TEMPSENSOR_MODULE       29
-#define ESP32_REGDMA_MODULE           30
-#define ESP32_LP_I2C0_MODULE          31
+#define ESP32_LEDC_MODULE                0
+#define ESP32_UART0_MODULE               1
+#define ESP32_UART1_MODULE               2
+#define ESP32_USB_MODULE                 3
+#define ESP32_I2C0_MODULE                4
+#define ESP32_I2S1_MODULE                5
+#define ESP32_TIMG0_MODULE               6
+#define ESP32_TIMG1_MODULE               7
+#define ESP32_UHCI0_MODULE               8
+#define ESP32_RMT_MODULE                 9
+#define ESP32_PCNT_MODULE                10
+#define ESP32_SPI_MODULE                 11
+#define ESP32_SPI2_MODULE                12
+#define ESP32_TWAI0_MODULE               13
+#define ESP32_TWAI1_MODULE               14
+#define ESP32_RNG_MODULE                 15
+#define ESP32_RSA_MODULE                 16
+#define ESP32_AES_MODULE                 17
+#define ESP32_SHA_MODULE                 18
+#define ESP32_ECC_MODULE                 19
+#define ESP32_HMAC_MODULE                20
+#define ESP32_DS_MODULE                  21
+#define ESP32_SDIO_SLAVE_MODULE          22
+#define ESP32_GDMA_MODULE                23
+#define ESP32_MCPWM0_MODULE              24
+#define ESP32_ETM_MODULE                 25
+#define ESP32_PARLIO_MODULE              26
+#define ESP32_SYSTIMER_MODULE            27
+#define ESP32_SARADC_MODULE              28
+#define ESP32_TEMPSENSOR_MODULE          29
+#define ESP32_ASSIST_DEBUG_MODULE        30
+/* LP peripherals */
+#define ESP32_LP_I2C0_MODULE             31
+#define ESP32_LP_UART0_MODULE            32
 /* Peripherals clock managed by the modem_clock driver must be listed last */
-#define ESP32_WIFI_MODULE             32
-#define ESP32_BT_MODULE               33
-#define ESP32_IEEE802154_MODULE       34
-#define ESP32_COEX_MODULE             35
-#define ESP32_PHY_MODULE              36
-#define ESP32_MODULE_MAX              37
+#define ESP32_WIFI_MODULE                33
+#define ESP32_BT_MODULE                  34
+#define ESP32_IEEE802154_MODULE          35
+#define ESP32_COEX_MODULE                36
+#define ESP32_PHY_MODULE                 37
```

Comment:
How come that the values are suddenly changing? Was this a bug here or in ESP-IDF? I'm asking because we are currently struggling with weird interrupt and clock setup issues for the IEEE 802.15.4 driver and this may be related.

Reply:
The values here can change according to HAL version, depending on new peripheral support. The value is mapped in this function: https://github.com/zephyrproject-rtos/hal_espressif/blob/zephyr/components/hal/esp32c6/include/hal/clk_gate_ll.h#L21